packages/react/src/components/Tabulation/index.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+class Tabulation extends Component {
+  constructor(props) {
```

Comment:
What about using hooks here to simplify the overall codebase?

Reply:
Could be. You good for pushing that for next iteration on the components?

Reply:
If you're not familiar enough with hooks, I can handle it (if it helps 🙂)